llvm-runtimes/libatomic-stub/libatomic-stub-1.0.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Stub library which allows compiler-rt to be used as an libatomic"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+S="${WORKDIR}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	llvm-runtimes/compiler-rt[atomic-builtins]
```

Comment:
Needs to be `atomic-builtins(-)` (if the USE flag doesn't exist on a version, the functionality isn't there) to fix the pkgcheck warnings.

(In future, please do ask explicitly if you don't know how to fix a warning.)